gulpfile.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-use-before-define */
```

Comment:
is this necessary? what's it for?

Reply:
The `gulp.task` lines are before the actual task functions. It might be easier to find the available tasks when they are defined first, so I did not want to move them down.

Reply:
Sounds good